host_os.py

```diff
@@ -34,6 +35,7 @@
     'build-package': build_package,
     'release-notes': create_release_notes,
     'upgrade-versions': upgrade_versions,
+    'update-versions-readme': update_versions_readme,
```

Comment:
I'll send a patch in another PR which renames upgrade-versions/upgrade_versions to update-versions/update_versions. Pls let us keep this inconsistency for now.